Dockerfile

```diff
@@ -1,21 +1,39 @@
 FROM docker.io/library/alpine:3.19
 
+ARG AUTOCONF_VERSION="2.71"
```

Comment:
When consider our release pipeline it should be a 2.72 no?

Reply:
My suggestion would be:

1. host this container on a repo in the `python` GitHub org
2. switch CI and build system to use the container
3. upgrade to 2.72

This is why I think the default should still be 2.71.

Reply:
OTOH, those steps are pretty much a single step, so it does perhaps not matter 😆

Reply:
Ah got it :) Sounds reasonable.